### PR TITLE
fix: 🐞 window is not defined

### PR DESCRIPTION
**Pull request by theryansmee**
Add null check for window

✅ Closes: #225

## PR Checklist

Please check if your PR fulfills the following requirements:

- [x] The commit message follows our guidelines: https://github.com/ngneat/falso/blob/master/CONTRIBUTING.md#commit
- [x] Tests for the changes have been added (for bug fixes / features)
- [x] Docs have been added / updated (for bug fixes / features)

## PR Type

What kind of change does this PR introduce?

```
[x] Bugfix
[ ] Feature
[ ] Code style update (formatting, local variables)
[ ] Refactoring (no functional changes, no api changes)
[ ] Build related changes
[ ] CI related changes
[ ] Documentation content changes
[ ] Other... Please describe:
```

## What is the current behavior?

Issue Number: N/A

## Does this PR introduce a breaking change?

```
[ ] Yes
[x] No
```

### CODE REVIEW

File: docs/src/consolePlayground/index.js [hunk 1]
```diff
@@ -1,10 +1,12 @@
 import * as falso from '../theme/ReactLiveScope/falso.min';
 
 window.onload = function () {
-	const titleStyles = 'color:#8a76d9; font-size: 22px; font-weight:bold;';
-	const copyStyles = 'color:white; font-size: 14px';
-	const exampleStyles = 'color:#8a76d9; font-size: 14px;';
-	console.log('%cFalso\n\n' + '%cYou can use _ to access Falso functions here in the console. \n\n%c' + 'For example: _.randFullName()', titleStyles, copyStyles, exampleStyles)
+	if (typeof window) {
```
Comment (NetanelBasal) [inline docs/src/consolePlayground/index.js:4]: if (typeof window !== 'undefined')

Comment (theryansmee) [reply docs/src/consolePlayground/index.js]: 😬 Sorry mate

**Review by NetanelBasal — APPROVED** [review]

File: docs/src/consolePlayground/index.js (hunk 1 above)
Comment (theryansmee) [reply docs/src/consolePlayground/index.js]: 3rd time lucky 🤞

Comment (NetanelBasal) [reply docs/src/consolePlayground/index.js]: You're right, missed this. I'm with Covid these days 😂

Comment (NetanelBasal) [reply docs/src/consolePlayground/index.js]: @theryansmee wrap the entire block and merge, please

**Pull request merged**